src/endpointRoutes.js

```diff
@@ -0,0 +1,122 @@
+'use strict'
+
+const KoaBodyParser = require('@viweei/koa-body-parser')
+
+const logger = require('./logger')
+const {handleServerError} = require('./util')
+const {
+  deleteEndpoint,
+  createEndpoint,
+  updateEndpoint
+} = require('./db/services/endpoints')
+
+const createEndpointRoute = router => {
+  router.post('/endpoints', KoaBodyParser(), async (ctx, next) => {
+    const failureMsg = 'Endpoint creation/update failed: '
+
+    try {
+      const body = Object.assign({lastUpdated: Date.now()}, ctx.request.body)
+
+      await createEndpoint(body)
+        .then(result => {
+          ctx.status = 201
+          ctx.body = result
+          logger.info(
+            `Endpoint "${result.endpoint.name}" has been successfully created on endpoint ${result.endpoint.pattern}`
```

Comment:
@bradsawadye the word successfully seems redundant next to the word created as `successfully` is implied

```suggestion
            `Endpoint "${result.endpoint.name}" created on ${result.endpoint.pattern}`
```